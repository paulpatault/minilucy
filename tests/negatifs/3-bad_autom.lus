node main0 (lo, hi : int) returns (x: int);
let
  automaton
  | Chaud ->
      x = 0 -> pre x + 1;
      until (x >= hi) continue Froid
  | Froid ->
      x = 0 -> pre x - 1;
      until (x <= lo) continue Chaud2
  end
tel
