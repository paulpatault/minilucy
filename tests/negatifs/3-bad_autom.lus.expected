[31m[1mSugar syntax error: [0mundefined constructor Chaud2 in automaton

